src/styles/blocks/card.scss

```diff
@@ -0,0 +1,117 @@
+.card {
+  width: 200px;
+  height: fit-content;
+  box-sizing: border-box;
+  margin: 0;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+
+  &__img  {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info-area {
+    margin: 40px 0 0;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    color: #060b35;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    letter-spacing: 0;
+    text-align: left;
+    justify-content: left;
+    margin: 0 0 4px;
+    padding: 0;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    letter-spacing: 0;
+    text-align: left;
+    justify-content: left;
+    color: #616070;
+    margin-bottom: 16px;
+  }
+
+  &__review-area {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
+
+  &__review-text {
+    font-size: 10px;
+    font-weight: 400;
+    letter-spacing: 0;
+    text-align: right;
+  }
+
+  &__price-area {
+    display: flex;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    letter-spacing: 0;
+    text-align: center;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-title {
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+    letter-spacing: 0;
+    text-align: left;
+    display: inline;
+    margin: 0;
+    color: #616070;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    letter-spacing: 0;
+    text-align: right;
+    display: inline;
+  }
+
+  &__button {
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    background-color: #00ACDC;
+    color: #fff;
+    text-decoration: none;
+    line-height: 40px;
+    justify-content: center;
+    font-size: 14px;
+    font-weight: 700;
+    letter-spacing: 0;
+    text-align: center;
+    margin: 16px 0 0;
+    padding: 0;
+
+    &:hover {
+      border: 1px solid #00ACDC;
+      background-color: #fff;
+      color: #00ACDC;
+      box-sizing: border-box;
+    }
```

Comment:
All hover transitions should be fluid both going in and out of hover, can you think of a way to implement that?

Reply:
Hi @EdPirro, I put transition: all 0.5s; in the new commit. Let me know if you have more adjustments.